composer.json

```diff
@@ -22,15 +22,14 @@
         ]
     },
     "require": {
-        "symfony/framework-bundle": "2.*",
-        "symfony/console": "2.*",
+        "symfony/framework-bundle": "^2.7|^3.0",
```

Comment:
You are correct, but composer will just install the old version of this bundle which still are compatible.